dice.py

```diff
@@ -40,21 +51,17 @@ def roll(self, count, sides):
 
 		return response
 
-	def receive(self, msg):
-		coarse = msg.body.strip().split(" ")
-		cmd = coarse[0]
-		rolls = list(filter( lambda x: re.match(die_re, x), coarse[1:] ))
+	def _err(self):
```

Comment:
I think `_err` for this function name is too generic, as there is at least one other error case when the number of sides is 0

Reply:
probably makes more sense to make this a string constant anyway